com.unity.netcode.gameobjects/Runtime/Messaging/MessagingSystem.cs

```diff
@@ -59,6 +60,8 @@ public SendQueueItem(NetworkDelivery delivery, int writerSize, Allocator writerA
 
         internal Type[] MessageTypes => m_ReverseTypeMap;
         internal MessageHandler[] MessageHandlers => m_MessageHandlers;
```

Comment:
SortedList may be a more appropriate solution to this problem?

Reply:
Yeah. I went with a "Not redoing the design" approach.